src/components/keyboard/keyboard.js

```diff
@@ -184,85 +189,99 @@ class Keyboard extends Component {
         const from = this.props.player.currentTime;
         const to = 0;
         this.props.logger.debug(`Seek. ${from} => ${to}`);
         this.props.player.currentTime = to;
         this.props.updateOverlayActionIcon(IconType.StartOver);
         this.toggleHoverState();
-        return {from: from, to: to};
+        return {preventDefault: true, payload: {from: from, to: to}};
       }
+      return {preventDefault: true, payload: null};
     },
-    [KeyMap.END]: () => {
+    [KeyMap.END]: (): KeyboardEventResult => {
       if (!(this.props.player.ads && this.props.player.ads.isAdBreak()) && !(this.props.player.isLive() && !this.props.player.isDvr())) {
         const from = this.props.player.currentTime;
         const to = this.props.player.duration;
         this.props.logger.debug(`Seek. ${from} => ${to}`);
         this.props.player.currentTime = to;
         this.props.updateOverlayActionIcon(IconType.SeekEnd);
         this.toggleHoverState();
-        return {from: from, to: to};
+        return {preventDefault: true, payload: {from: from, to: to}};
       }
+      return {preventDefault: true, payload: null};
     },
-    [KeyMap.M]: () => {
+    [KeyMap.M]: (): KeyboardEventResult => {
       this.props.logger.debug(this.props.player.muted ? 'Umnute' : 'Mute');
       this.props.player.muted = !this.props.player.muted;
       this.props.player.muted
         ? this.props.updateOverlayActionIcon([IconType.VolumeBase, IconType.VolumeMute])
         : this.props.updateOverlayActionIcon([IconType.VolumeBase, IconType.VolumeWaves]);
-      return true;
+      return {preventDefault: true, payload: true};
     },
-    [KeyMap.SEMI_COLON]: (event: KeyboardEvent) => {
-      if (event.shiftKey && this.props.player.playbackRate !== this.props.player.defaultPlaybackRate) {
-        this.props.logger.debug(`Changing playback rate. ${this.props.player.playbackRate} => ${this.props.player.defaultPlaybackRate}`);
-        this.props.player.playbackRate = this.props.player.defaultPlaybackRate;
-        this.props.updateOverlayActionIcon(IconType.Speed);
-        return {speed: this.props.player.defaultPlaybackRate};
+    [KeyMap.SEMI_COLON]: (event: KeyboardEvent): KeyboardEventResult => {
+      if (event.shiftKey) {
+        if (this.props.player.playbackRate !== this.props.player.defaultPlaybackRate) {
+          this.props.logger.debug(`Changing playback rate. ${this.props.player.playbackRate} => ${this.props.player.defaultPlaybackRate}`);
+          this.props.player.playbackRate = this.props.player.defaultPlaybackRate;
+          this.props.updateOverlayActionIcon(IconType.Speed);
+          return {preventDefault: true, payload: {speed: this.props.player.defaultPlaybackRate}};
+        }
+        return {preventDefault: true, payload: null};
       }
+      return {preventDefault: false, payload: null};
```

Comment:
if `{preventDefault: false, payload: null}` repeats everywhere extract it to a const and use that.
`const unhandledKeyboardEventResult = {preventDefault: false, payload: null};`